test/checkpoint.bats

```diff
@@ -143,3 +143,52 @@ function teardown() {
 	[[ "$container_name" == "restored-sleep-container" ]]
 	[[ "$pod_name" == "restoresandbox2" ]]
 }
+
+@test "checkpoint and try to restore one container with an open TCP connection and expected failure" {
+	# This test needs to fail during restore to see if the log file is saved at a location
+	# the user can access it.
+	# This test is starting a container and creates an established TCP connections.
```

Comment:
A simpler way to simulate an error could be by adding unrecognised option to the CRIU configuration file just before restore.